docs/topics/gradle/gradle-kotlin-dsl.md

```diff
@@ -0,0 +1,48 @@
+[//]: # (title: Using the Gradle Kotlin DSL)
+
+The Gradle Kotlin DSL is a domain specific language to write build scripts in more convenient way.
```

Comment:
This is the new sentence. Everything ahead is old except the "What's next" section.
We can add more details about the DSL later if we decide to do so.